src/en/proposals/dragonjspider-Wawaspecies.md

```diff
@@ -0,0 +1,64 @@
+This doc will go into detail about their stats and abilities, and discussing the possible impacts some of these may have in different scenarios.
+Feedback would be greatly appreciated; as there are also some experimental ideas that may be unbalanced, or unfun to play around.
+
+## Background
+We have discussed this a lot on the discord, leading to this proposal (starting from https://discord.com/channels/310555209753690112/1212937078153805844/1220355368228618310). There have been a lot of ideas behind this, and there may be more that were overlooked.
+Feel free to read all of it, but this doc will also summarise what were agreed to be good ideas, as well as the more experimental/system effecting changes. We also understand that there may be some copyright issues with this, so I will attach any permissions to this as well, If and when any are given.
+
+## Copyright permissions
+Currently we have permission from a dev of Downpour, expeditions specifically. Whilst this isn't "proper" approval, more of a semi-approval, we believe it allows us to continue with a project of making a "Scug Inspired" species. The following screenshots are provided courtesy of fairlysadpanda.
+
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/ae7437cf-f4d3-42c1-8798-b8ce24a2725f)
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/69d00309-faae-41e0-a6aa-9e2916052de3)
+
+
+
+## Species
+Our Official name for the species would be Wawa (proposed by fairlysadpanda). Whilst this isn't exactly "creative" for them, we believe a distinct but still related name suits them. We're not going to be naming them Scug(s), or Slugcat(s), because of more specific issues with copyright and such, and lack of explicit written permission.
+For personalization they will be able to have most colors for skin, similar to lizards or moths.
+Anything unmentioned will be similar to humans, like having two hands or a backpack slot.
+
+The conventional naming scheme for them, while mostly a suggestion, would be as follows: Number, Object. E.G: Seven-Hundred Nukies, Twenty-Eight Wounds, Seventy-Three Bombs. (suggested by kiterny)
+Alternatively, they could use a single word describing their character, similar to how players name lizards do, but I felt this might make them too similar in character to lizards. 
+Or finally, it could be pet names, which may be too far off what would normally be acceptable, but may be considered for names. (because of course people would name these things silly names and stuff)
+
+## Abilities
+- They will deal additional piercing damage with spears (+4 damage with spears) (suggested by thecatbastard, cornerdogs, dragonjspider)The following is the time it would take to crit a human with no resists: 
```

Comment:
Specifically doing more damage with spears seems kind of odd, but I wouldn't *mind* having it.

Reply:
Well I guess we (being everyone who added to the conversation in the initial post on discord) wanted to keep *some* aspect from the game, spears was a decent option without adding an insane amount of code or being overpowered.